### PR TITLE
Cow Mutantrace

### PR DESCRIPTION
**Pull request by mybluecorners**
this adds the cow-human mutantrace into the game. It is a bioeffect and it is also spacebux purchaseable. It comes with special sprites, speech sound, and talk sound.

cowrp never dies

### CODE REVIEW

File: code/datums/mutantraces.dm [hunk 1]
```diff
@@ -1563,6 +1563,97 @@
 			src.real_name = "RELIQUARY TEST - DO NOT USE"
 			src.set_mutantrace(/datum/mutantrace/reliquary_soldier)
 
+/datum/mutantrace/cow
+	name = "cow"
+	// icon_state = "lizard"
```
Comment (ZeWaka) [inline code/datums/mutantraces.dm:1568]: -_-

File: code/datums/mutantraces.dm [hunk 2]
```diff
@@ -1563,6 +1563,97 @@
...
+	proc/fix_colors(var/hex)
+		var/list/L = hex_to_rgb_list(hex)
+		for (var/i in L)
+			L[i] = min(L[i], 190)
+			L[i] = max(L[i], 50)
+		if (L.len == 3)
+			return rgb(L["r"], L["g"], L["b"])
+		return rgb(22, 210, 22)
+
+	disposing()
```
Comment (ZeWaka) [inline code/datums/mutantraces.dm:1600]: umm uselesssss?????

File: code/datums/mutantraces.dm (hunk 1 above)
Comment (flrsh) [inline code/datums/mutantraces.dm]: Probably remove?

**Comment by mybluecorners** [comment]
you were too fast for your own good

File: code/datums/mutantraces.dm (hunk 1 above)
Comment (mybluecorners) [reply code/datums/mutantraces.dm]: i already resolved that, also no fake cows allowed

**Pull request merged**